packages/terra-form-select/tests/wdio/select-spec.js

```diff
@@ -1,10 +1,4 @@
 Terra.describeViewports('Select', ['tiny'], () => {
-  describe('should render an empty placeholder', () => {
-    before(() => browser.url('/#/raw/tests/terra-form-select/form-select/empty-placeholder'));
-
-    Terra.it.validatesElement();
-  });
-
   describe('should render a placeholder', () => {
```

Comment:
Should this be removed as well since placeholder isn't a thing anymore? or does this test and test page need to be reamed to reflect what is being tested here?

Reply:
removed it here https://github.com/cerner/terra-core/pull/3216/commits/23e9a1ec3c0f197cdad5aa97352efa8a5cf5e5df